core/web-assets/src/main/assets/static/load-assets.jsp

```diff
@@ -60,7 +65,7 @@ if (locator == null) {
                         .append(resource.getPath())
                         .append("?v=").append(lastModified)
                         .append("\"");
-                    if (mediaString != null && mediaString.trim() != "") {
+                    if (mediaString != null && mediaString.trim().equals("")) {
```

Comment:
the second half of this should be negated: `if (mediaString != null && !mediaString.trim().equals("")) {`

Reply:
:man_facepalming: thanks @RangerRick